File: Creational/Lab4/Python/estate_to_sell_factory.py
from estate_factory import *

class EstateToSellFactory(EstateFactory):
    def create_estate(self, type: str) -> RealEstate:
        e = self.estate_from_type(type)
        e.is_buyable = False
        e.is_rentable = False
        e.is_sellable = True
        return e

File: Creational/Lab4/C#/Factories/EstateToSellFactory.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace C_.Factories
{
    public class EstateToSellFactory : EstateFactory
    {
        public override RealEstate CreateEstate(string type)
        {
            var e = EstateFromType(type);
            e.IsBuyable = false;
            e.IsRentable = false;
            e.IsSellable = true;
            return e;
        }
    }
}
